src/SapiEmitter.php

```diff
@@ -46,62 +45,26 @@ public function __construct(?int $bufferSize = null)
         $this->bufferSize = $bufferSize ?? self::DEFAULT_BUFFER_SIZE;
     }
 
-    /**
-     * Responds to the client with headers and body.
-     *
-     * @param ResponseInterface $response Response object to send.
-     * @param bool $withoutBody If body should be ignored.
-     *
-     * @throws HeadersHaveBeenSentException
-     */
-    public function emit(ResponseInterface $response, bool $withoutBody = false): void
```

Comment:
Have you kept `withoutBody` behavior somehow?

Reply:
I split one `emit()` method to two: `emit()` and `emitHeaders()` (see `EmitterInterface`). When we need to send only headers, we should call `emitHeaders()` instead `emit()`.